tests/testthat/test-MAEFilterStates.R

```diff
@@ -3,7 +3,7 @@ testthat::test_that("The constructor does not throw", {
     input_dataname = "test",
     output_dataname = "test",
     datalabel = character(0),
-    varlabels = character(0),
+    varlabels = "varlabel1",
```

Comment:
I will comment it here as a remind, please revert these changes. We should stay with the default for the old tests.

Reply:
done in new commit